src/Misc/SegmentedProgressBar.tsx

```diff
@@ -0,0 +1,126 @@
+import React, { ReactElement, useCallback } from 'react';
+import styled, { keyframes } from 'styled-components';
+
+const TooltipText = styled.span<{width?: string}>`
+  visibility: hidden;
+  padding: 5px;
+  font-size: 10px;
+  color: #000000;
+  line-height: 1.5;
+  text-align: left;
+  border: 1px solid hsl(195deg 5% 60% / 72%);
```

Comment:
Could you please update the border to a more common border?
border: 1px solid ${({theme: {colors}}) => colors.divider};